pyrokinetics/gene.py

```diff
@@ -372,6 +380,25 @@ def load_numerics(self, pyro, gene):
 
         numerics = Numerics()
 
+        # Set number of fields
+        numerics.phi = True
+
+        try:
+            if gene['general']['beta'] > 0:
+                numerics.apar = True
+            else:
+                numerics.apar = False
+        except KeyError:
+            numerics.apar = False
+
+        try:
+            if gene['general']['bpar']:
+                numerics.bpar = True
+            else:
+                numerics.bpar = False
+        except KeyError:
+            numerics.bpar = False
```

Comment:
This pattern of `try... except KeyError` can be replaced with [`dict.get()`](https://docs.python.org/3/library/stdtypes.html#dict.get):

```suggestion
        numerics.apar = gene['general'].get('beta', 0) > 0
        numerics.bpar = gene['general'].get('bpar', False)
```